RecoLocalTracker/SubCollectionProducers/src/JetCoreClusterSplitter.cc

```diff
@@ -64,13 +76,45 @@ class JetCoreClusterSplitter : public edm::stream::EDProducer<> {
   double centralMIPCharge_;
 };
 
+void JetCoreClusterSplitter::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+  edm::ParameterSetDescription desc;
+
+  desc.add<std::string>("pixelCPE");
+  desc.add<bool>("verbose");
+  desc.add<double>("ptMin");
+  desc.add<double>("deltaRmax");
+  desc.add<double>("chargeFractionMin");
+  desc.add<edm::InputTag>("pixelClusters");
+  desc.add<edm::InputTag>("vertices");
+  desc.add<edm::InputTag>("cores");
+  desc.add<double>("forceXError");
+  desc.add<double>("forceYError");
+  desc.add<double>("fractionalWidth");
+  desc.add<double>("chargePerUnit");
+  desc.add<double>("centralMIPCharge");
```

Comment:
please add defaults for all parameters, otherwise we won't be able to parse them in confDB.  @cms-sw/hlt-l2 FYI